app/models/status_message.rb

```diff
@@ -121,6 +89,13 @@ def post_location
     }
   end
 
+  def people_allowed_to_be_mentioned
```

Comment:
This method was replaced with [this line](https://github.com/diaspora/diaspora/pull/6818/files#diff-1b42e0e2c5d972f2a8ef8ea55b6b92dfR23) after rebase on 46cbc6e52a9d4a03dff8228ce11ac4876d247a15, so it's not used. Thanks for noticing.